### PR TITLE
[ResOps] Create PR & Issue Templates

### LINKED ISSUE #1: Setup Repository Issue & PR Templates
## Feature Description

Create a `feature` and `bug` issue template and a `pull_request` template

## Problem You're Trying to Solve

Keeping all the issues created in a similar format with all the required data will help the scrum masters get a better overview of the project status.

## Related External Resources

- [Issue Template Docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)
- [PR Template Docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)

- https://github.com/stevemao/github-issue-templates
- https://github.com/devspace/awesome-github-templates

### PR DESCRIPTION
**Pull request by MNThomson**
# Description

Added the PR/Issue templates to enforce a consistent schema

Closes #1

## Checklist
- [x] The code includes tests if relevant
- [x] I have *actually* self-reviewed my changes and done QA

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1_feature_request.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+name: 🚀 Feature Request
```
Comment (sejal-agarwal) [inline .github/ISSUE_TEMPLATE/1_feature_request.md:2]: There's emojis!!!! 😍

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Description
+<!-- Please include a summary of the change(s) and which issue is being fixed. Please provide as much detail as possible. -->
+
+
+<!-- Replace `XXX` with the concerning issue number. The "#" links this PR to its relevant issue -->
```
Comment (kylestang) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Do we want to explicitly say that PRs *must* have a related issue?

Comment (sejal-agarwal) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: I would prefer if PRs have a related issue. If a PR doesn't have a related issue, can't they just remove the "Closes #XXX"?

Comment (TobySMurray) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: Yeah I think any PR should have a ticket associated with it

**Review by TobySMurray — APPROVED** [review]
LGTM!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (MNThomson) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: Created #2 to enforce links to an issue

**Pull request merged**